src/sentry/static/sentry/app/components/events/meta/annotatedText.jsx

```diff
@@ -114,7 +114,7 @@ const AnnotatedText = ({value, chunks, errors, remarks, ...props}) => (
 );
 
 AnnotatedText.propTypes = {
-  value: PropTypes.string,
+  value: PropTypes.any,
```

Comment:
when this file is converted to typescript, this type will be React.ReactNode

Reply:
In that case, should this be `PropTypes.node` instead?